modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,102 @@
+* A label key must start with a lowercase letter.
+* You can configure a maximum of 32 labels per resource. Each resource can have a maximum of 64 labels, and 32 labels are reserved for internal use by {product-title}.
+
+.Limitations for user-defined labels
+
+* Labels for `Compute Address` are supported in the GCP beta version. {product-title} does not add labels to the resource.
+
+User-defined labels and {product-title} labels are seen on the following GCP resources:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+.User-defined tags
+
+User-defined `tags` are attached to resources created by {product-title} core operator `cluster-image-registry-operator` and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by the foloowing operators are not tagged:
+* Installer Operator
+* Machine API Provider GCP operator 
+* GCP Filestore CSI Driver Operator
+* `GCP PD CSI Driver Operator
+
+The following are the requirements for the user-defined tags:
+
+* Tag key and tag value must already exist. {product-title} does not create the key and the value .
+* A tag `parentID` can be either `OrganizationID` or `ProjectID`:
+** An `OrganizationID` must consist of decimal numbers, and cannot have leading zeros.
+** A `ProjectID` must be 6 to 30 characters in length, can only contain lowercase letters, numbers, and hyphens, and must start with a letter, and cannot end with a hyphen.
+* A tag key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A tag key must contain only uppercase and lowercase alphanumeric characters, hyphen (`-`), underscore (`_`), and period (`.`).
+* A tag value must contain only uppercase and lowercase alphanumeric characters, hyphen (`-`), underscore (`_`), period (`.`), at sign (`@`), percent sign (`%`), equals sign (`=`), plusse (`+`), colon (`:`), comma (`,`), asterisk (`*`), pound sign (`$`), ampersand (`&`), parentheses (`()`), square braces (`[]`), curly braces (`{}`), and space.
+* A tag key and value must begin and end with an alphanumeric character.
+* Tag value must be one of the pre-defined values for the key.
+* You can configure a maximum of 50 tags.
+* There should be no tag key defined with the same value as any of the existing tag keys that will be inherited from the parent resource.
+
+//Cluster creation phase
+include::installing-gcp-cluster-creation.adoc[leveloffset=+1]
+
+.Limitations for the user-defined tags
+
+* Tags will not be attached to the following items:
+** Control plane instances and storage buckets created by the installation program
+** Compute instances created by the Machine API provider for GCP
+** Filestore instance resources created by the GCP Filestore CSI Driver Operator
+** Compute disk and compute image resources created by the GCP PD CSI Driver Operator
```

Comment:
This is a duplicate content

Reply:
I re-arranged the initial content but, some operators may be are similiar to the labels.

Reply:
Lines 53-57 sound similar. Either they are same or overlapped. Both are related to tags. They could be placed together.

Reply:
Removed the redundant parts.